src/image_modifications.cpp

```diff
@@ -1067,6 +1105,70 @@ REGISTER_MOD_PARSER(XBRZ, args)
 	return std::make_unique<xbrz_modification>(factor);
 }
 
+// PAD
+REGISTER_MOD_PARSER(PAD, args)
+{
+	int top = 0;
+	int right = 0;
+	int bottom = 0;
+	int left = 0;
+
+	// Check for the presence of an '=' sign to determine the parsing mode.
+	if(args.find('=') != std::string_view::npos) {
+
+		// --- Keyword-Argument Mode ---
+		const auto params = utils::map_split(std::string{args}, ',', '='); // map_split needs a std::string
+
+		// Map valid input strings to the corresponding integer reference
+		static const std::map<std::string, int*> alias_map = {
+			{"top", &top},
+			{"t", &top},
+			{"right", &right},
+			{"r", &right},
+			{"bottom", &bottom},
+			{"b", &bottom},
+			{"left", &left},
+			{"l", &left},
+		};
+
+		// Parse and assign values if keywords are valid
+		for(const auto& [key, value] : params) {
+			auto it = alias_map.find(key);
+			if(it != alias_map.end()) {
+				utils::optional<int> result = utils::from_chars<int>(value);
+				if(result) {
+					*it->second = result.value();
```

Comment:
Should be consistent with the code below.

```suggestion
				if(utils::optional<int> padding = utils::from_chars<int>(value)) {
					*it->second = padding.value();
```

Looks like `utils::optional` (without the `<int>`) works as well. Feel free to change both cases to that if you like.

Reply:
done